packages/mantine/src/components/search/Search.tsx

```diff
@@ -0,0 +1,14 @@
+import {SearchSize16Px} from '@coveord/plasma-react-icons';
+import {ActionIcon} from '@mantine/core';
+
+export interface SearchProps {
+    handleSearch: () => void;
+}
+
+const SearchButton: React.FunctionComponent<SearchProps> = ({handleSearch}) => (
```

Comment:
Until we have a formal request for it, I would prefer having one less component to maintain, document and support. We can safely keep this one internal to the CodeEditor until further notice.

Reply:
I understand, I moved the search folder inside of code-editor 👍